### DISCUSSION TITLE
On which version of PHP are you running Adminer ?

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [adminerevo/adminerevo#25] [Polls]
As we are thinking to deprecate support for very old versions of PHP, we would like first to have a view on the current install base to make the good decision.
More answers = better decision, please answer if you use Adminer !

**Comment by Lumeriol** [comment]
In my opinion, AdminerEvo should remain with full support for old versions of PHP until the end of life of version 4.x.
Version 4.x is there almost 10 years with too much old code and create any good extension or find problem is not easy (i. e. DISTINCT function for now).

However, from version 5 it would be a good idea to stay with PHP 8.1+ only.
For most things, Adminer is currently fully sufficient, PHP version 5.x is long dead, version 7 has update support no longer, and version 8.0 ends in November.

Not to mention that if someone wants to contribute to development, they won't to think about whether they can use specific feature or bypass it with pofyfil.

But version 4.x should be supported for a while, with bug fixes and minor improvements that could be ported to it.

**Comment by mckaygerhard** [comment]
php 5.6 and 7.x must be supported until end of 32bit OSs on 2034!

**Reply by Lumeriol** [reply]
Yes, let these old versions be supported for another 10 years, but in the version of the application designed for it. So there may well be some final version of 4.9.x that will support the old PHP versions, but development is continuously moving forward and there is no point in creating new features with pointless complexity for backwards compatibility. 

Plus, there's nothing stopping anyone from backporting these features into a fork/PR of the old version.

**Comment by BertrandB21** [comment]
Hello PHP8.2 FPM (Debian Trixie)  but got a problem with pdo.sqlite not working so need to add sqlite in query
